api/bnd.bnd

```diff
@@ -2,9 +2,23 @@
     org.eclipse.microprofile.*
 
 Import-Package: \
-    javax.enterprise.util;version="[1.1,3)", \
+    javax.enterprise.*;-remove-attribute:=version, \
+    javax.inject;-remove-attribute:=version,\
     *
 
+Require-Capability: \
+    osgi.contract;\
+        osgi.contract=JavaCDI;\
+        filter:="(&(osgi.contract=JavaCDI)(version=${cdi.api.version}))",\
```

Comment:
Which bundle provide this capability? The cdi jar from maven central does not, which is why my comment below.

Reply:
we've made this requirement effective:=active in order to prevent runtime failure.